governance/votes/Executive vote - August 8, 2024.md

```diff
@@ -0,0 +1,116 @@
+- **Proposal**: [Forum post](https://forum.makerdao.com/t/bounty-payout-request-for-immunefi-bug-32005/24605)
+
+If this executive proposal passes, then the following payments as part of the bug bounty program will be distributed.
+
+- **100,000 DAI** will be transferred to [0xA4a6B5f005cBd2eD38f49ac496d86d3528C7a1aa](https://etherscan.io/address/0xA4a6B5f005cBd2eD38f49ac496d86d3528C7a1aa).
+- **10,000 DAI** will be transferred to [0x7119f398b6C06095c6E8964C1f58e7C1BAa79E18 (immunefi.eth)](0x7119f398b6C06095c6E8964C1f58e7C1BAa79E18).
+
+A technical explanation of the rationale for the bug bounty payment can be found in this [forum comment](https://forum.makerdao.com/t/bounty-payout-request-for-immunefi-bug-32005/24605/3).
+
+### ClipperMomJob Update
+
+- **Authorization**: [Forum post](https://forum.makerdao.com/t/executive-inclusion-clippermomjob-update/24774/2)
+- **Proposal**: [Forum post](https://forum.makerdao.com/t/executive-inclusion-clippermomjob-update/24774/)
+
+If this executive proposal passes, the Keeper Network's ClipperMomJob will be updated through the following contract calls:
+
+- `sequencer.removejob(0xc3a76b34cfbda7a3a5215629a0b937cbdec7c71a)`.
+- `sequencer.addjob(0x7E93C4f61C8E8874e7366cDbfeFF934Ed089f9fF)`.
+
+The chainlog entry for CRON_CLIPPER_MOM_JOB will also be updated to the new address. The new ClipperMomJob is deployed at [0x7E93C4f61C8E8874e7366cDbfeFF934Ed089f9fF](https://etherscan.io/address/0x7E93C4f61C8E8874e7366cDbfeFF934Ed089f9fF) and the code is available on GitHub [here](https://github.com/makerdao/dss-cron/tree/clipper-mom-try-all-classes).
+
+### Aligned Delegate Compensation
+
+- **Authorization**: [$link_to_approval]()
+- **Proposal**: [$link_to_proposal]()
+
+If this executive proposal passes, then $executive_entry_3_implications.
+
+### 6S Capital [Stability Fee](https://mips.makerdao.com/mips/details/MIP104#14-3-1-3-stability-fee-sf-) Increase
+
+- **Authorization**: [Forum post](https://forum.makerdao.com/t/rwa-001-6s-capital-update-and-stability-fee-proposal/24624/2)
+- **Proposal**: [Forum post](https://forum.makerdao.com/t/rwa-001-6s-capital-update-and-stability-fee-proposal/24624/)
+
+If this executive proposal passes, the 6S Capital (RWA001-A) Stability Fee will be increased as follows:
+
+- Increase the RWA001-A Stability Fee by 6 percentage points from 3% to **9%**.
+
+### Monetalis Clydesdale and Coinbase Custody DAO Resolutions
+
+- **Authorization**: [$link_to_approval]()
+- **Proposal**: [$link_to_proposal]()
+
+If this executive proposal passes, the following DAO Resolution authorizing the actions needed for the winddown of the Monetalis Clydesdale and Coinbase Custody facilities will be approved:
+
+- Approve Monetalis Clydesdale and Coinbase Custody DAO Resolution with IPFS hash [QmaYKt61v6aCTNTYjuHm1Wjpe6JWBzCW2ZHR4XDEJhjm1R](https://ipfs.io/ipfs/QmaYKt61v6aCTNTYjuHm1Wjpe6JWBzCW2ZHR4XDEJhjm1R).
+

```

Comment:
```suggestion

Additional details on the winddown process for the Clydesdale Vault and the Coinbase Custody Vault can be found in this [forum discussion](https://forum.makerdao.com/t/clydesdale-vault-hq/17923/88).
```

Reply:
Unless the forum link is the only one we will have, in which case it will go to the "Proposal".